docs/tutorials/02_vibrational_structure.ipynb

```diff
@@ -0,0 +1,761 @@
+      "- 1.7817866707286076 * IIIIIIIIZXIX\n",
+      "- 11.899156895555553 * IIIIIIIIIYIY\n",
+      "+ 8.124377387683996 * ZIIIIIIIIYIY\n",
+      "+ 4.874626432610398 * IZIIIIIIIYIY\n",
+      "+ 1.6248754775367993 * IIZIIIIIIYIY\n",
+      "+ 11.28726712478577 * IIIZIIIIIYIY\n",
+      "+ 6.772360274871463 * IIIIZIIIIYIY\n",
+      "+ 2.2574534249571543 * IIIIIZIIIYIY\n",
+      "- 8.90893335364304 * IIIIIIZIIYIY\n",
+      "- 5.345360012185823 * IIIIIIIZIYIY\n",
+      "- 1.7817866707286076 * IIIIIIIIZYIY\n",
+      "- 2.2979209375000007 * XIXIIIIIIXIX\n",
+      "- 2.2979209375000007 * YIYIIIIIIXIX\n",
+      "- 2.2979209375000007 * XIXIIIIIIYIY\n",
+      "- 2.2979209375000007 * YIYIIIIIIYIY\n",
+      "- 3.1925212500000018 * IIIXIXIIIXIX\n",
+      "- 3.1925212500000018 * IIIYIYIIIXIX\n",
+      "- 3.1925212500000018 * IIIXIXIIIYIY\n",
+      "- 3.1925212500000018 * IIIYIYIIIYIY\n",
+      "- 44.54543265324755 * IIIIIIXXIXIX\n",
+      "- 44.54543265324755 * IIIIIIYYIXIX\n",
+      "- 44.54543265324755 * IIIIIIXXIYIY\n",
+      "- 44.54543265324755 * IIIIIIYYIYIY\n",
+      "+ 2.5198268750000015 * IIIIIIXIXXIX\n",
+      "+ 2.5198268750000015 * IIIIIIYIYXIX\n",
+      "+ 2.5198268750000015 * IIIIIIXIXYIY\n",
+      "+ 2.5198268750000015 * IIIIIIYIYYIY\n",
+      "- 31.498377500000007 * IIIIIIIXXXIX\n",
+      "- 31.498377500000007 * IIIIIIIYYXIX\n",
+      "- 31.498377500000007 * IIIIIIIXXYIY\n",
+      "- 31.498377500000007 * IIIIIIIYYYIY\n",
+      "- 2.102429985900245 * XXIIIIIIIIXX\n",
+      "- 2.102429985900245 * YYIIIIIIIIXX\n",
+      "- 2.102429985900245 * XXIIIIIIIIYY\n",
+      "- 2.102429985900245 * YYIIIIIIIIYY\n",
+      "+ 0.8980418750000001 * IXXIIIIIIIXX\n",
+      "+ 0.8980418750000001 * IYYIIIIIIIXX\n",
+      "+ 0.8980418750000001 * IXXIIIIIIIYY\n",
+      "+ 0.8980418750000001 * IYYIIIIIIIYY\n",
+      "+ 1.175575188382615 * IIIXXIIIIIXX\n",
+      "+ 1.175575188382615 * IIIYYIIIIIXX\n",
+      "+ 1.175575188382615 * IIIXXIIIIIYY\n",
+      "+ 1.175575188382615 * IIIYYIIIIIYY\n",
+      "- 0.5021409375000001 * IIIIXXIIIIXX\n",
+      "- 0.5021409375000001 * IIIIYYIIIIXX\n",
+      "- 0.5021409375000001 * IIIIXXIIIIYY\n",
+      "- 0.5021409375000001 * IIIIYYIIIIYY\n"
+     ]
+    }
+   ],
+   "source": [
+    "vibrational_problem = VibrationalStructureProblem(driver, num_modals=3, truncation_order=2)\n",
+    "second_q_ops = vibrational_problem.second_q_ops()\n",
+    "\n",
+    "qubit_converter = QubitConverter(mapper = DirectMapper())\n",
+    "\n",
+    "qubit_op = qubit_converter.convert(second_q_ops[0])\n",
+    "\n",
+    "print(qubit_op)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now that the Hamiltonian is ready, it can be used in a quantum algorithm to find information about the vibrational structure of the corresponding molecule. Check out our tutorials on Ground State Calculation and Excited States Calculation to learn more about how to do that in Qiskit!"
```

Comment:
> Check out our tutorials on Ground State Calculation and Excited States Calculation to learn more about how to do that in Qiskit!"

Maybe, now that Qisklit Nature is its own package and has more of a standalone identity, though still carrying the Qiskit name of course, maybe we should be saying `...about how to do that in Qiskit Nature!` 

I made the comment on this one place but I saw it above as well. Maybe this comment applies to other tutorials here too.